client/i18n/locales/english/intro.json

```diff
@@ -2732,28 +2732,29 @@
         "title": "Build an Exercise Tracker",
         "intro": ["In this lab, you will build an exercise tracker"]
       },
-      "tbei": { "title": "376", "intro": [] },
-      "kegi": { "title": "377", "intro": [] },
-      "quiz-backend-javascript": {
-        "title": "Backend JavaScript Quiz",
-        "intro": [
-          "Test what you've learned in this quiz on Backend JavaScript."
-        ]
-      },
-      "vnfu": { "title": "379", "intro": [] },
-      "rlli": { "title": "380", "intro": [] },
-      "rjpv": { "title": "381", "intro": [] },
-      "xbri": { "title": "382", "intro": [] },
-      "kwhp": { "title": "383", "intro": [] },
-      "nkjt": { "title": "384", "intro": [] },
-      "pgjj": { "title": "385", "intro": [] },
-      "exom": { "title": "386", "intro": [] },
-      "bvqc": { "title": "387", "intro": [] },
-      "xish": { "title": "388", "intro": [] },
-      "front-end-development-certification-exam": {
-        "title": "Front End Development Certification Exam",
-        "intro": ["", ""]
+      "lab-file-metadata-microservice": {
```

Comment:
Is there a reason why the diff is so large here?

Because you are only replacing one title, so I am not sure why it is showing this many changes

Was there something else added here?

Reply:
I messed up something, now should be fine